pkg/server/app/sort.go

```diff
@@ -0,0 +1,21 @@
+package app
+
+import "sort"
+
+type ByKey []*EnvVar
```

Comment:
maybe `EnvVarsByKey`?

Reply:
I think that `ByKey` is more cleaner, like in sort documentation

Reply:
but think about usave 
```go
import "pkg/client/app/sort"
...
sort.ByKey
```
`ByKey` of what?

Reply:
you can use the func `sortEnvsByKey`